# Households comparison - 2050

Pairwise comparison of total households at **2050** between two forecasts, by control/target area and grouped by county.

- **A**: Summer 2026 (pipeline)
- **B (baseline)**: Legacy LUVit (RTP26) (spreadsheet)

`% Diff = (A - B) / B * 100`.

In [1]:
from comparison_helpers import compare_hh_target_area, compare_hh_region

FORECAST_A = 'summer_2026'
FORECAST_B = 'legacy_luvit_rtp26'  # baseline (denominator)
YEAR = '2050'

## Region

In [2]:
compare_hh_region(FORECAST_A, FORECAST_B, YEAR)

,Summer 2026,Legacy LUVit (RTP26),Diff,% Diff
King,"1,251,820","1,287,442","-35,622",-2.8%
Pierce,"458,269","500,454","-42,185",-8.4%
Snohomish,"499,833","481,881","+17,952",+3.7%
Kitsap,"137,512","150,061","-12,549",-8.4%
Region,"2,347,434","2,419,838","-72,404",-3.0%


## King County

In [3]:
compare_hh_target_area(FORECAST_A, FORECAST_B, YEAR, 53033)

,Summer 2026,Legacy LUVit (RTP26),Diff,% Diff
Bellevue,"100,675","100,738",-63,-0.1%
Seattle,"458,691","485,162","-26,471",-5.5%
Auburn,"40,369","40,308",+61,+0.2%
Bothell,"18,320","18,549",-229,-1.2%
Burien,"28,586","28,921",-335,-1.2%
Federal Way,"48,600","48,943",-343,-0.7%
Issaquah,"18,966","20,152","-1,186",-5.9%
Kent,"57,201","57,964",-763,-1.3%
Kirkland,"52,570","53,247",-677,-1.3%
Redmond,"50,992","50,671",+321,+0.6%


## Pierce County

In [4]:
compare_hh_target_area(FORECAST_A, FORECAST_B, YEAR, 53053)

,Summer 2026,Legacy LUVit (RTP26),Diff,% Diff
Tacoma,"133,992","149,610","-15,618",-10.4%
Auburn,"3,731","4,125",-394,-9.6%
Lakewood,"36,422","39,509","-3,087",-7.8%
Puyallup,"25,690","27,912","-2,222",-8.0%
University Place,"20,107","22,196","-2,089",-9.4%
DuPont,"6,185","6,345",-160,-2.5%
Fife,"6,340","6,576",-236,-3.6%
Fircrest,"3,723","4,013",-290,-7.2%
"Mid-County, Parkland-Spanaway-Midland, South Hill","76,651","78,078","-1,427",-1.8%
Sumner,"6,661","7,237",-576,-8.0%


## Snohomish County

In [5]:
compare_hh_target_area(FORECAST_A, FORECAST_B, YEAR, 53061)

,Summer 2026,Legacy LUVit (RTP26),Diff,% Diff
Everett,"89,271","84,973","+4,298",+5.1%
Bothell,"15,082","14,089",+993,+7.0%
Lynnwood,"31,282","29,785","+1,497",+5.0%
Arlington,"16,752","15,138","+1,614",+10.7%
Bothell MUGA,"18,053","17,554",+499,+2.8%
Edmonds,"29,096","27,177","+1,919",+7.1%
Edmonds MUGA,"2,154","2,135",+19,+0.9%
Everett MUGA,"27,765","26,384","+1,381",+5.2%
Larch Way Overlap,"5,087","4,392",+695,+15.8%
Lynnwood MUGA,"26,536","24,939","+1,597",+6.4%


## Kitsap County

In [6]:
compare_hh_target_area(FORECAST_A, FORECAST_B, YEAR, 53035)

,Summer 2026,Legacy LUVit (RTP26),Diff,% Diff
Bremerton,"25,186","28,926","-3,740",-12.9%
Bremerton UGA,"5,421","5,910",-489,-8.3%
Silverdale,"13,430","13,914",-484,-3.5%
Bainbridge Island,"12,588","13,257",-669,-5.0%
Kingston UGA,"2,878","3,207",-329,-10.3%
Port Orchard,"10,937","11,535",-598,-5.2%
Port Orchard UGA,"7,337","7,966",-629,-7.9%
Poulsbo,"6,747","7,712",-965,-12.5%
Poulsbo PUTA,646,703,-57,-8.1%
Central Kitsap UGA,"11,875","12,038",-163,-1.4%


## Notes

- `% Diff` is computed as `(A - B) / B * 100`, where **B** (baseline) is the
  second forecast listed at the top of the notebook.
- Cell color: red when `|% Diff| >= 5%`, amber when `>= 2.5%`, green otherwise.
- Target areas present in only one forecast still appear, with `-` for the
  missing forecast's value.